test/helpers/data/workload-capacity-helper.js

```diff
@@ -135,6 +128,29 @@ module.exports.addWorkloadCapacitiesForOffenderManager = function () {
       ids.forEach((id) => {
         inserts.push({table: 'workload_points_calculations', id: id})
       })
+
+      var defaultTier = {
+        workload_id: inserts.filter((item) => item.table === 'workload')[0].id,
+        tier_number: 1,
+        overdue_terminations_total: 10,
+        unpaid_work_total: 10,
+        warrants_total: 10,
+        total_cases: 10,
+        location: 'COMMUNITY'}
+
+      var tiers = []
+
+      for (let i = 1; i <= 7; i++) {
+        tiers.push(Object.assign({}, defaultTier, {tier_number: i, location: 'COMMUNITY'}))
+        tiers.push(Object.assign({}, defaultTier, {tier_number: i, location: 'CUSTODY'}))
+        tiers.push(Object.assign({}, defaultTier, {tier_number: i, location: 'LICENSE'}))
+      }
```

Comment:
I'm not seeing the value of adding so many tiers. I know it's more like the real data but I don't think we need it all to prove that query works. We could probably get away with just 2 tier rows and verify that the values we expect are summed correctly. It would keep things similar in the helper and reduce database load.